### DISCUSSION TITLE
Guidelines to conduct Pixel to pixel segmentation (Medical Image Segmentation) without using bounding boxes using Detectron library.

### DISCUSSION THREAD

**Discussion by Ritam-M** [facebookresearch/detectron2#3635] [Q&A]

**Comment by prdai** [comment]
hi,
https://github.com/facebookresearch/detectron2/issues/1055

the above is a similar issue

**Comment by prdai** [comment]
https://colab.research.google.com/github/TannerGilbert/Object-Detection-and-Image-Segmentation-with-Detectron2/blob/master/Detectron2_train_on_a_custom_dataset.ipynb#scrollTo=ypm-lLy-wxdN

**Comment by prdai** [comment]
> https://colab.research.google.com/github/TannerGilbert/Object-Detection-and-Image-Segmentation-with-Detectron2/blob/master/Detectron2_train_on_a_custom_dataset.ipynb#scrollTo=ypm-lLy-wxdN

this is the tutorial
